content/en/service_management/workflows/variables.md

```diff
@@ -0,0 +1,148 @@
+
+All the variables of the Source object are visible in the Context Variables tab.
+
+{{< img src="service_management/workflows/context-variables-tab-source-object-variables2.png" alt="The Source object variables in the Context Variables tab" style="width:60%;">}}
+
+## Input parameters
+
+Input parameters are immutable key-value pairs that you can use to pass data into a workflow. You can use input parameters in workflows that:
+- are triggered manually, such as from a Dashboard.
+- use mention triggers, such as Monitors and Security Signal Notification Rules.
+
+To add an input parameter:
+1. Click on the workflow canvas.
+1. Click the **+** icon next to **Input Parameters**.
+1. Add a parameter name, data type, and description for the parameter. The display name is generated automatically from the parameter name. Check the **Use custom display name** box to customize it. The display name is a human readable name for the parameter, while the parameter name is used to reference the parameter in your workflow steps.
+1. Optionally, add a default value for the parameter. If you add a default value, the parameter is optional at runtime.
+
+To reference the input parameter in a step, use the syntax `{{ Trigger.<parameter name>}}`. For example, to reference an input parameter named `user`, use `{{Trigger.user}}`.
+
+The **Input Parameters** section displays the names of all existing input parameters together with a counter. Hover over a counter to see which steps are using the parameter.
+
+{{< img src="service_management/workflows/input-parameter3.png" alt="Hover over a counter to see which steps are using the parameter." style="width:60%;">}}
+
+You can add an implicit input parameter (a parameter that doesn't already exist in the workflow) by typing it into a workflow step using the `{{ Trigger.<parameter name> }}` syntax. The next time you save the workflow, a dialog appears allowing you to convert the parameter to an explicit parameter. For more information on triggering workflows, see [Trigger a workflow][5].
+
+If you're looking for an existing input parameter, start typing `{{ Trigger.` to see if it appears as a suggestion. Alternatively, consult the [Context Variables](#context-variables) tab for a list of available parameters.
+
+## Output parameters
+
+You can pass the result of a workflow using output parameters.
+
+To add an output parameter:
+1. Click on the workflow canvas.
+1. Click the **+** icon next to **Output Parameters**.
+1. Add a parameter name, value, and data type for the parameter.
+1. Optionally, add a default value for the parameter. If you add a default value, the parameter is optional at runtime.
+
+The **Output Parameters** section displays the names of all existing output parameters together with a counter.
+
```

Comment:
```suggestion
For information on using output parameters with workflows, see [Trigger a workflow from a workflow][7].
```
Need to add a small section to this page for that. I can follow up with a commit